pkg/gits/git_cli.go

```diff
@@ -456,6 +456,16 @@ func (g *GitCLI) HasChanges(dir string) (bool, error) {
 	return len(text) > 0, nil
 }
 
+// HasFileChanged indicates if there are any changes to a file in the repository from the given directory
+func (g *GitCLI) HasFileChanged(dir string, fileName string) (bool, error) {
+	text, err := g.gitCmdWithOutput(dir, "status", "-s", fileName)
```

Comment:
I think `--porcelain` would be the safer option, since that is guaranteed to be fixed and can even be pinned by specifying a version. I guess `-s` will work for now though.